vue/components/forms/personalization-form/reference.vue

```diff
@@ -208,6 +211,8 @@ export const Reference = {
                     this.tabProperties.forEach(propertyName => {
                         const property = this.propertiesData[group][propertyName];
                         if (!property) return;
+                        const availableProperties = this.properties()[propertyName] || [];
+                        if (availableProperties.length < 2) return;

```

Comment:
Comment this. Something like "if there is a single option don't include the properties in the tab message"

Reply:
... otherwise in 1 week we'll forget why did we add this here :p